.changeset/rotten-bananas-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Adds color and color theme for use in Learn Copilot branding.
```

Comment:
Can we rewrite the description to be not Learn Copilot branding specific as I don't think users will know what that means? Maybe just "Add gradient vivid colors"?